lib/router/model-tasks.js

```diff
@@ -8,7 +8,7 @@ module.exports = taskflow => {
   Case.db(taskflow.db);
 
   router.param('modelId', (req, res, next, modelId) => {
-    req.modelId = modelId;
+    req.modelId = /^\d+$/.test(modelId) ? parseInt(modelId) : modelId;
```

Comment:
Always add a base to a `parseInt` call. Otherwise odd things might occur (they won't in _this_ case, but it's good bulletproof socks)

```suggestion
    req.modelId = /^\d+$/.test(modelId) ? parseInt(modelId, 10) : modelId;
```